src/parcel/AdapterJSAsset.js

```diff
@@ -39,9 +39,14 @@ class AdapterJSAsset extends JSAsset {
   }
 
   async getPackage() {
-    const pkg = await super.getPackage();
-    pkg.devDependencies.hyperapp = '*';
-    return pkg;
+    const pkg = await super.getPackage() || {};
+    const pack = pkg.devDependencies ? pkg : Object.assign(pkg, {
+      devDependencies: {
+        hyperapp: '*',
```

Comment:
this is set twice now...

Reply:
Only when `devDependencies` didn't exist before.

Reply:
why do you need it, btw ?

Reply:
In order to get `parcel` to think that the pragma for JSX files is `h` instead of `React.createElement` without forcing people to add it to their JSX files.